src/routes/posts/+page.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+	import ImagePages from "$lib/components/imagepages.svelte";
+	import { parseContent } from "$lib/parse";
+
+	export let data;
+	$: articles = data.articles;
```

Comment:
Yeah, I think because of how articles are paged (navigated using next and back buttons at the bottom). The page depends on a query parameter, which when changed makes the page server load another set of articles. So whenever articles changes, what's displayed on the page also need to change

Reply:
Does changing the `$:` to a `let` change the behavior? I feel like if the server reruns the load function, then you shouldn't need it to be reactive.

Reply:
you are so true, data.articles works